From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Samuel Attard <samuel@Samuels-Mac.local>
Date: Thu, 2 Jul 2020 19:03:46 -0700
Subject: remove some deps that do not work on arm64


diff --git a/.vpython b/.vpython
index 0a28e76064296edea609eb52e57555689c23c0a3..edd43d286bdc14c165cbc90cbf7da64620c266e4 100644
--- a/.vpython
+++ b/.vpython
@@ -53,10 +53,10 @@ wheel: <
 #
 # This version must be compatible with the version range specified by
 # //third_party/catapult/telemetry/telemetry/internal/util/external_modules.py
-wheel: <
-  name: "infra/python/wheels/psutil/${vpython_platform}"
-  version: "version:5.2.2"
->
+#wheel: <
+#  name: "infra/python/wheels/psutil/${vpython_platform}"
+#  version: "version:5.2.2"
+#>
 
 # Used by:
 #   third_party/catapult
@@ -94,10 +94,10 @@ wheel: <
 #
 # This version must be compatible with the version range specified by
 # //third_party/catapult/telemetry/telemetry/internal/util/external_modules.py
-wheel: <
-  name: "infra/python/wheels/numpy/${vpython_platform}"
-  version: "version:1.11.3"
->
+#wheel: <
+#  name: "infra/python/wheels/numpy/${vpython_platform}"
+#  version: "version:1.11.3"
+#>
 
 # Used by:
 #   third_party/catapult
@@ -132,10 +132,10 @@ wheel: <
 
 # Used by:
 #   testing/buildbot/generate_buildbot_json_coveragetest.py
-wheel: <
-  name: "infra/python/wheels/coverage/${vpython_platform}"
-  version: "version:4.3.4"
->
+#wheel: <
+#  name: "infra/python/wheels/coverage/${vpython_platform}"
+#  version: "version:4.3.4"
+#>
 
 # Used by:
 #   build/android/pylib/local/emulator/avd.py
@@ -268,10 +268,10 @@ wheel <
   name: "infra/python/wheels/funcsigs-py2_py3"
   version: "version:1.0.2"
 >
-wheel: <
-  name: "infra/python/wheels/psutil/${vpython_platform}"
-  version: "version:5.2.2"
->
+#wheel: <
+#  name: "infra/python/wheels/psutil/${vpython_platform}"
+#  version: "version:5.2.2"
+#>
 
 # Used by:
 #   tools/infra
